sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -331,7 +334,8 @@ def shutdown(self):
 
 class SdkWorker(object):
 
-  def __init__(self, bundle_processor_cache, profiler_factory=None):
+  def __init__(self, bundle_processor_cache,
+               profiler_factory=None):
```

Comment:
unintended change?

Reply:
Intended (separation of the kwargs from the positional args).